### PR TITLE
Added support for ItemPhysic (again)

### PR DESCRIPTION
**Pull request by CreativeMD**
Might not be the best prettiest solution. (We used it in 1.12).
If you have suggestions on how to make this better just let me know.

### CODE REVIEW

**Comment by HellFirePvP** [comment]
Here we go again :P

File: src/main/java/hellfirepvp/astralsorcery/common/entity/item/EntityItemHighlighted.java [hunk 1]
```diff
@@ -33,9 +36,21 @@
...
+        try {
+            skipPhysicRenderer = ObfuscationReflectionHelper.findField(ItemEntity.class, "skipPhysicRenderer");
+        } catch (UnableToFindFieldException e) {}
+    }
 
     public EntityItemHighlighted(EntityType<? extends ItemEntity> type, World world) {
         super(type, world);
+        if(skipPhysicRenderer != null)
+            try {
+                skipPhysicRenderer.setBoolean(this, true);
```
Comment (HellFirePvP) [inline src/main/java/hellfirepvp/astralsorcery/common/entity/item/EntityItemHighlighted.java:52]: use ObfuscationHelper.setPrivateValue() instead; so we don't need to try/catch ourselves.

Comment (CreativeMD) [reply src/main/java/hellfirepvp/astralsorcery/common/entity/item/EntityItemHighlighted.java:52]: Bad for performance, the field should be cached.

Comment (HellFirePvP) [reply src/main/java/hellfirepvp/astralsorcery/common/entity/item/EntityItemHighlighted.java:52]: Derp. my bad. Thought for a moment it took a Field not just the fieldname.

**Comment by yavincl** [comment]
By the way, why is this being merged into 1.16-indev instead of master?

**Comment by CreativeMD** [comment]
ups

**Comment by Wolfstar1010** [comment]
Hello! Thanks for working on this! I saw that the issue was in the changelog for the most recent Astral update but I'm still having the issue occur in game after updating. Is there something special I'd have to do to fix this?

**Comment by CreativeMD** [comment]
Have you update ItemPhysic as well?

**Comment by Wolfstar1010** [comment]
Yes, I just updated both.

**Comment by Wolfstar1010** [comment]
I have Item Physics 1.4.11 and Astral Sorcery 1.13.5

**Comment by yavincl** [comment]
Maybe the fix didn't work, or uhh the fix was not properly merged

**Comment by CreativeMD** [comment]
My bad, should be fixed in the newest version of ItemPhysic.

**Comment by Wolfstar1010** [comment]
:( Sorry to be back here again, I'm still having the same issue. I wish I could easily share a video of what's going on... but here is a picture. The items placed are laying flat and frantically rotating around a central point. Please let me know if I'm doing something wrong... I updated both Astral and Item Physics just now before launching...

image

image

**Pull request merged**